api/server/router/swarm/cluster_routes.go

```diff
@@ -265,6 +273,29 @@ func (sr *swarmRouter) getServiceLogs(ctx context.Context, w http.ResponseWriter
 	return nil
 }
 
+func (sr *swarmRouter) getTaskLogs(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
+	if err := httputils.ParseForm(r); err != nil {
+		return err
+	}
+
+	// (dperny) i think it's safe to modify the vars/request like this but if
+	// there are REAL WEIRDO bugs later on down the line, y'all can double
+	// check here
+	r.Form.Add("tasks", vars["id"])
+	// TODO(dperny) i might should delete the id from vars before handling
+	// this request? but i don't think it matters
+	return sr.getSwarmLogs(ctx, w, r, vars)
```

Comment:
Ok, then my second question (for you and anyone else) is where is the best place to put the `getSwarmLogs` function, if it does not conform to the `Handler` function type? Can I leave it in the `cluster_routes.go` file? I believe explicitly should NOT go in the daemon, I think the pattern of having the daemon construct the logs byte stream response is Bad and i'm addressing that in a separate PR.